tests/Unit/GeoRouteTest.php

```diff
@@ -64,13 +64,20 @@ public function testDefaultCallback()
             ->once()
             ->andReturn(json_decode('{"countryCode": "ca"}'));
 
-        $this->assertEquals(401, $this->get('/foo')->getStatusCode());
+        try {
+            $response = $this->get('/foo');
+        } catch (\Symfony\Component\HttpKernel\Exception\HttpException $e) {
+            $this->assertSame(401, $e->getStatusCode());
+            $exceptionThrown = true;
```

Comment:
Since you only define the $exceptionThrown variable here, won't you get an error if it doesn't fail? (missing variable)

Reply:
Fixed in new commit